File: tree-sitter-dart/test/corpus_smoke/06_control.dart
String classify(int n) {
  if (n < 0) {
    return 'negative';
  } else if (n == 0) {
    return 'zero';
  }
  var acc = 0;
  for (var i = 0; i < n; i++) {
    acc += i;
  }
  while (acc > 100) {
    acc ~/= 2;
  }
  do {
    acc--;
  } while (acc > 50);
  switch (n % 3) {
    case 0:
      return 'fizz';
    case 1:
      return 'one $acc';
    default:
      return 'other';
  }
}

File: tree-sitter-zig/test/corpus_smoke/06_control.zig
fn classify(n: i32) []const u8 {
    if (n < 0) {
        return "negative";
    } else if (n == 0) {
        return "zero";
    }
    var i: usize = 0;
    while (i < 10) : (i += 1) {
        if (i == 5) break;
    }
    for ([_]u8{ 1, 2, 3 }, 0..) |v, idx| {
        _ = v;
        _ = idx;
    }
    return switch (n) {
        1...9 => "digit",
        else => "big",
    };
}
